Dockerfile

```diff
@@ -3,8 +3,9 @@ MAINTAINER David Personette <dperson@gmail.com>
 
 # Install tor and privoxy
 RUN apk --no-cache --no-progress upgrade && \
-    apk --no-cache --no-progress add bash curl privoxy shadow tini tor tzdata&&\
+    apk --no-cache --no-progress add bash curl privoxy shadow tini tor tzdata && \
     file='/etc/privoxy/config' && \
+    touch $file && \
```

Comment:
This command just creates an empty file.

The following `sed` commands are doing nothing after that. So after this fix `privoxy` will not function as intended by the maintainer.